app/models/defence_request.rb

```diff
@@ -10,8 +10,14 @@ class DefenceRequest < ActiveRecord::Base
   scope :not_draft, -> { where.not(state: "draft") }
   scope :ordered_by_created_at, -> { order(created_at: :asc) }
 
-  def self.has_solicitor(solicitor)
-    where(solicitor_uid: solicitor.uid)
+  class << self
```

Comment:
Think deleting has_solicitor() is a good idea. I'll check first that it is not used elsewhere.